scss/detail.scss

```diff
@@ -0,0 +1,40 @@
+$gray : #828282;
+
+.book {
+  display: flex;
+  font-family: 'Roboto', sans-serif;
+  flex-direction: column;
+  padding: 15px;
+}
+
+.row {
+  padding: 18px 0;
```

Comment:
Same. Try to define this into a more specific class like `book-title` instead of doing it in a reusable class. One tip

Use padding to separate parent tags from their children and margin to separate siblings.